doc/data/messages/c/consider-using-join/bad.py

```diff
@@ -0,0 +1,6 @@
+def fruits_to_string():
+    fruits = {"apple", "pear", "peach"}
+    formatted_fruit = ""
+    for fruit in fruits:
+        formatted_fruit += fruit  # [consider-using-join]
+    return formatted_fruit
```

Comment:
```suggestion
def fruits_to_string(fruits):
    formatted_fruit = ""
    for fruit in fruits:
        formatted_fruit += fruit  # [consider-using-join]
    return formatted_fruit

print(fruits_to_string(["apple", "pear", "peach"]))
```
List/tuples are better for iterating :)